client/src/Poem.jsx

```diff
@@ -24,6 +25,8 @@ const Poem = ({ poem }) => {
       <div className="poemText">
         {poemLineNodes}
       </div>
+      <CommentBox url="/api/comments" key={poem.id} poemId={poem.id} pollInterval={2000} />
```

Comment:
Is the `key` attribute here necessary? I thought that was only if a component is repeated within another component.